### DISCUSSION TITLE
这个组件库和coloruiBeta有什么区别？看demo像是同一个？

### DISCUSSION THREAD

**Discussion by x011223** [Color-UI/MP-CU#27] [Q&A] [answered]
https://github.com/weilanwl/coloruiBeta

如果微信小程序/企业微信小程序推荐使用哪个。望解惑，谢谢。

**Comment by iZaiZaiA** [comment, marked as answer]
这个是微信原生小程序使用的，coloruiBeta是uni-app使用的，coloruiBeta可以用uni-app打包多个平台，但mp-cu，只能用于微信原生小程序开发。针对的开发群体不一样。
